src/biotite/structure/info/atoms.py

```diff
@@ -18,7 +18,7 @@
 # fmt: on
 
 
-def residue(res_name):
+def residue(res_name, allow_missing_coords=False):
```

Comment:
Just a nit: Throughout Biotite 'coordinates' is abbreviated with `coord`, so I suggest to use it also here

```suggestion
def residue(res_name, allow_missing_coord=False):
```